tests/urlpatterns/tests.py

```diff
@@ -208,7 +209,7 @@ class ConversionExceptionTests(SimpleTestCase):
     def test_resolve_value_error_means_no_match(self):
         @DynamicConverter.register_to_python
         def raises_value_error(value):
-            raise ValueError()
+            raise Http404()
```

Comment:
Please add a second test for converter that will raise `Http404`, also a test for technical response (in `view_tests/tests/test_debug.py`) when used converter returns `Http404`.